src/liberty/libertyProject.ts

```diff
@@ -132,38 +133,61 @@ export class ProjectProvider implements vscode.TreeDataProvider<LibertyProject>
 				}
 			}).catch((err: any) => console.error("Unable to parse build.gradle: " + gradlePath + "; " + err));
 		}
-
 		return validGradleBuildFiles;
 	}
 
+	private projectRootPathExists(path: string, keys: Iterable<string>): Boolean {
+		for (const existingPath of keys) {
+			if ( vscodePath.dirname(existingPath) === path) {
+				return true;
+			}
+		  }
+		return false;
+	}
+
+	/**
+	 * Checks if given build file exists in existing project map (<code>this.projects</code>).
+	 * If it exists, then the corresponding liberty project will be added to the given 
+	 * <code>projectsMap</code>.
+	 * 
+	 * @param buildFilePath The build file to check (full path to pom.xml or build.gradle)
+	 * @param projectType Project type
+	 * @param projectsMap Existing projects.
+	 * @returns 
+	 */
+	private addExistingProjectToNewProjectsMap (buildFilePath: string, projectType: string, projectsMap: Map<string, LibertyProject> ) : Boolean {
```

Comment:
Method rename suggestion for readability
```suggestion
	private isExistingProject(buildFilePath: string, projectType: string, projectsMap: Map<string, LibertyProject> ) : Boolean {
```

Reply:
@kathrynkodama I like the original name better, as we do add the project into the map.  I would like to keep the original method name.  Thanks